src/main/java/gregtech/loaders/recipe/chemistry/PolymerRecipes.java

```diff
@@ -470,4 +469,37 @@ private static void polybenzimidazoleProcess() {
                 .output(dust, ChromiumTrioxide, 4)
                 .buildAndRegister();
     }
+
+    public static void polycaprolactamProcess() {
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(400)
+                .fluidInputs(Benzene.getFluid(1000), Hydrogen.getFluid(6000))
+                .fluidOutputs(Cyclohexane.getFluid(1000))
+                .notConsumable(new IntCircuitIngredient(1))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[LV]).duration(100)
+                .fluidInputs(Chlorine.getFluid(2000), NitricOxide.getFluid(2000))
+                .fluidOutputs(NitrosylChloride.getFluid(2000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[MV]).duration(100)
+                .fluidInputs(Cyclohexane.getFluid(1000), NitrosylChloride.getFluid(1000))
+                .output(dust, Cyclohexanoxime, 19)
+                .fluidOutputs(HydrochloricAcid.getFluid(1000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(200)
+                .input(dust, Cyclohexanoxime, 19)
+                .fluidInputs(SulfuricAcid.getFluid(1000), Ammonia.getFluid(2000))
+                .output(dust, Caprolactam, 19)
+                .output(dust, AmmoniumSulfate, 15)
+                .buildAndRegister();
+
+        BLAST_RECIPES.recipeBuilder().EUt(VA[MV]).duration(1000).blastFurnaceTemp(533)
+                .input(dust, Caprolactam, 19)
+                .fluidInputs(Nitrogen.getFluid(1000))
+                .output(ingot, Polycaprolactam, 19)
+                .buildAndRegister();
```

Comment:
This recipe should be changed to `1 Caprolactam Dust + 1000 Nitrogen -> 1 Polycaprolactam Ingot` with a duration of `150` ticks.